src/main/java/com/github/manolo8/darkbot/utils/CommandLineUtils.java

```diff
@@ -0,0 +1,86 @@
+package com.github.manolo8.darkbot.utils;
+
+import com.github.manolo8.darkbot.gui.utils.Popups;
+
+import javax.swing.*;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class CommandLineUtils {
```

Comment:
This whole class is treated as static, it shouldn't. Consider naming it `StartupParams` or something like that, hold the specific info it needs, and pass it to main as a parameter. In general, avoid static when posslbe